pkg/runtime/runtime.go

```diff
@@ -2409,6 +2411,11 @@ func (a *DaprRuntime) processComponentAndDependents(comp componentsV1alpha1.Comp
 		timeout = defaultComponentInitTimeout
 	}
 
+	if compCategory == components.CategoryBindings && comp.Spec.Type == string(components.CategoryBindings)+".wasm" {
```

Comment:
I am not a fan of this TBH. I think we should modify the registry to indicate "this is a wasm component" without looking at the name.

What if we add more Wasm components in the future?

Reply:
Done, Add to those components marked as wasm. And wasm component should register themself.